bundle.go

```diff
@@ -120,39 +123,60 @@ func (b SortBundlesByVersion) Len() int           { return len(b) }
 func (b SortBundlesByVersion) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
 func (b SortBundlesByVersion) Less(i, j int) bool { return b[i].Version < b[j].Version }
 
+type SortBundlesByTime []Bundle
+
+func (b SortBundlesByTime) Len() int           { return len(b) }
+func (b SortBundlesByTime) Swap(i, j int)      { b[i], b[j] = b[j], b[i] }
+func (b SortBundlesByTime) Less(i, j int) bool { return b[i].Time.UnixNano() < b[j].Time.UnixNano() }
+
 // ValidateBundles is a plain validation type for a list of version bundles. A
 // list of version bundles is exposed by authorities. Lists of version bundles
 // of multiple authorities are aggregated and grouped to reflect distributions.
 type ValidateBundles []Bundle
 
+func (b ValidateBundles) Copy() ValidateBundles {
+	raw, err := json.Marshal(b)
```

Comment:
I don't know how to copy structs off the top of my head, but marshalling and unmarshalling them? Eh?

Reply:
Poor man's copy. When you have a better idea let me know. This is easy and works and we do not care about performance right now. I think the interface is there and implementation can just be improved later.

Reply:
Good enough for me.